cli.py

```diff
@@ -80,23 +77,31 @@ def attr_upd():
             return
 
         for ac_data in appliance_manager.aircons:
-            if ac_data["SAID"] == args.said:
-                await show_aircon_menu(backend_selector, auth, args.said, session)
+            if ac_data.said == args.said:
+                await show_aircon_menu(
+                    appliance_manager, backend_selector, auth, session, ac_data
+                )
```

Comment:
```suggestion
                await show_aircon_menu(ac_data)
```
Since the `Appliance` subclasses now get instantiated by the `ApplianceManager`, we can pass them to the menu modules. That way the cli will work the same way as home assistant, which is nicer for testing.
Also, we can move the `await manager.connect()` and `disconnect()` out of the menu modules and to here.

Reply:
Don't think this is going to work, unless you want to pass the aircon object to show_aircon_menu(), i.e.:

ac = Aircon(backend_selector, auth, session, app_data)
show_aircon_menu(ac)

Reply:
Don't think this is going to work, unless you want to pass the aircon object to show_aircon_menu(), i.e.:

ac = Aircon(backend_selector, auth, session, app_data)
show_aircon_menu(ac)

Reply:
Isn't `ac_data` an `Aircon` instance already? (we should probably rename it too, but not too important for this PR)

Reply:
Oh, duh!  Yes it is.  I'll resolve this as well.